naveenG/task2/edit.html

```diff
@@ -0,0 +1,114 @@
+    <title>Edit Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1><img src="clg1.jpg" width="6%" height="100px">NAVEEN COLLEGE OF ENGINEERING AND TECHNOLOGY,THENI-625531<img src="clg1.jpg" width="6%" height="100px"></h1>
+    <div class="color"><p class="color">Welcome to the registration page for alumni,Please take a moment to fill out the form below.</p>
+    <p class="color">Please supply all the requested information and click on SUBMIT,below.</p></div>
+     <form  method="get" target="_parent">
+        <div>
+            <label for="mem" >MEMBER OF INSTITUTE <span>*</span> :</label>
+            <input type="text" id="mem">
+
+        </div><br>
+        <div>
+            <label for="name">NAME <span>*</span> :</label>
+            <input type="text" id="name1" name="name1"  maxlength="20" placeholder="Name" >
+        </div><br>
+        <div>
+            <label for="name">FATHER NAME <span>*</span> :</label>
+            <input type="text" id="name2" name="name2" maxlength="20">
+        </div><br>
+        <div>
+            <label for="name">MOTHER NAME <span>*</span> :</label>
+            <input type="text" id="name3" name="name3"  maxlength="20">
+        </div><br>
+        <div>
+            <label for="year">YEAR OF LEAVING SCHOOL/COLLEGE <span>*</span> :</label>
+            <input type="text" id="completeyear"name="year"  maxlength="10">
+        </div><br>
+        <div>
+            <label for="add">ADDRESS <span>*</span> :</label>
+            <input type="text" id="adde" name="add"  maxlength="50" placeholder="Type address">
+        </div><br>
+        <div>
+            <label for="mail">EMAIL ID <span>*</span> :</label>
+            <input type="email" id="email" name="mail" maxlength="50" >
+        </div><br>
+        <div>
+            <label for="mobile">MOBILE NUMBER <span>*</span> :</label>
+            <input type="tel" id="mobile1" name="mobileno" maxlength="15" >
+        </div><br>
+        <div>
+            <label for="mobile">MOBILE NUMBER :</label>
+            <input type="tel" id="mobile" name="mobile" maxlength="15"  placeholder="optional">
+        </div><br>
+        <div>
+            <label for="status">PRESENT STATUS  :</label>
+            <input type="text" id="status" name="status" maxlength="50">
+        </div><br>
+        <div>
+            <label for="work">PRESENT WORKING PLACE :</label>
+            <input type="text" id="work" name="work" maxlength="50">
+        </div><br>
+        <div>
+            <label for="card">ID PROOF<span>*</span> :</label>
+            <input type="file" id="card" name="card"  maxlength="50">
+        </div><br>
+        <div>
+              <button type="submit" id="submit" class="button" onclick="">SUBMIT</button>
+                <button type="reset" class="button">RESET</button>
+          </div><br>
+        </div> 
+</form><hr>
+<script>
+function addrow()
```

Comment:
Move this Script code to separate JS file and provide link in the HTML page.
Indent the code properly in JS file.

Reply:
I gives the separate JS file for edit page and linked the HTML page and also I given the indent for JS code.